src/libraries/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.CurrentUserOnly.Unix.cs

```diff
@@ -26,15 +34,14 @@ public class NamedPipeTest_CurrentUserOnly_Unix
         public async Task Connection_UnderDifferentUsers_BehavesAsExpected(
             PipeOptions serverPipeOptions, PipeOptions clientPipeOptions)
         {
-            if (PlatformDetection.IsFedora)
-            {
-                // [ActiveIssue("https://github.com/dotnet/corefx/issues/38834")]
-                throw new SkipTestException("Failing on Fedora by not throwing expected exception");
-            }
 
             // Use an absolute path, otherwise, the test can fail if the remote invoker and test runner have
             // different working and/or temp directories.
             string pipeName = "/tmp/" + Path.GetRandomFileName();
+            bool isRoot = String.CompareOrdinal(Environment.UserName, "root") == 0;
```

Comment:
Nit, not need to change, but you can use `String.Equals(Environment.UserName, "root")`. It tests ordinal equality.

Reply:
```suggestion
            bool isRoot = Environment.UserName == "root");
```
But I think in some other tests we've used geteuid() == 0; is there a reason to prefer one over the other?

Reply:
geteuid() would be p/invoke or we can suck it in from Interop.
Environment.UserName is less efficient IMHO as it internally maps ID to a name. 
But it is a public API so it seems more natural to use it.

Reply:
Ok.  If the name being "root" is resilient, works for me.

Reply:
`root` is a magic name on Unix. It is possible to have multiple user names sharing the same ID or having `root` not really be a superuser. But that is certainly a pathological case and it will not happen during CI or normal use.   In this case, sudo will lookup 'root' and new logic will work as the current running and remote process owners will be the same.